tests/testthat/test-exrcise.R

```diff
@@ -0,0 +1,22 @@
+test_that("Exrcise transforms correctly", {
```

Comment:
Are you planning on adding any Rmd with fenced blocks or would that not be a helpful test?

Reply:
I assume you mean here non-chunk blocks like this:
~~~
```r
somecode()
```
~~~

This would be a good test to add. I started with some minimal tests, but I did confirm in informal testing that these are handled properly.

Reply:
Yep, that's what I meant 👍

Reply:
Added to test file. https://github.com/AlexsLemonade/exrcise/pull/4/commits/8361c0572ccb91890fb9183b0d0cb865c3a0c01f